.travis.yml

```diff
@@ -1,19 +1,16 @@
 sudo: false
 language: node_js
 node_js:
-  - "6"
+  - "14"
 
 cache:
   directories:
     - node_modules
 
 install:
-  - npm install -g yarn
-  - yarn install
+  - npm install gatsby@2.20.36
 
 script:
-  - npm run deploy:dev
-  - npm run deploy:prod
+  - npm run build
+
```

Comment:
Let's add another item here for running the linters, since that will make sure that CI is always checking whether code passes the lint check. You'll need to add another item to the `scripts` section of `package.json` to actually run ESLint. If you want to match what we named them in askdarcel-web, they're `lint` and `lint:fix`: https://github.com/ShelterTechSF/askdarcel-web/blob/17bcd0b7c65f7b8cfbe8522e42a6966ed420d3bc/package.json#L88-L89

Reply:
I added the lint scripts. I also added `public/` to the .eslintignore.